src/interface/viewport.h

```diff
@@ -41,7 +41,7 @@ enum {
 	VIEWPORT_FLAG_HIDE_BASE = (1 << 12),
 	VIEWPORT_FLAG_HIDE_VERTICAL = (1 << 13),
 	VIEWPORT_FLAG_INVISIBLE_SPRITES = (1 << 14),
-	VIEWPORT_FLAG_15 = (1 << 15)
+	VIEWPORT_FLAG_HIDE_TOP_BOTTOM_TOOLBAR = (1 << 15),
```

Comment:
Are you sure this flag can be simply replaced? Isn't it used for another function?